README.md

```diff
@@ -343,29 +415,48 @@ The following parts of these two libraries are exported:
 
 ### Deepseq
 
-`deepseq` is exported. For instance, if you want to force deep evaluation of
-some value (in `IO`), you can write `evaluateNF a`. WHNF evaluation is possible
-with `evaluateWHNF a`.
+All the main parts of the `deepseq` library are exported.
+For instance, if you want to force the deep evaluation of
+some value (in `IO`), you can write `evaluateNF a`.
+Additionally, the WHNF evaluation is possible
+with provided `evaluateWHNF`.
+
+## What's new?
 
-## What's new? [↑](#structure-of-this-tutorial)
+[[Back to the Table of Contents] ↑](#structure-of-this-tutorial)
 
-Finally, we can move to part describing the new cool features we bring with `relude`.
+Finally, let's move to part describing the new cool features we bring with
+`relude`.
 
 ### Available by default
 
-* Safe analogue for list functions: use `viaNonEmpty` function to get `Maybe a`.
-  + `viaNonEmpty head :: [a] -> Maybe a`
+* Safe analogue for the list functions:
+  ```haskell
+  head :: NonEmpty a -> a
```

Comment:
I thought it would be cool to specify all such functions in here.
```suggestion
  head :: NonEmpty a -> a    -- the first element of the list
  tail :: NonEmpty a -> [a]  -- the list without the first element
  last :: NonEmpty a -> a    -- the last element of the list
  init :: NonEmpty a -> [a]  -- the list without the last element
```